src/stories/Library/disclosure/disclosure.scss

```diff
@@ -82,4 +82,11 @@
       transform: scaleY(-1);
     }
   }
+
+  &.disclosure--no-header-padding .disclosure__headline {
```

Comment:
In this case I think it makes more sense defining a modifier on the headline element.
So something like:
```
&__headline {
     xyz
     &.disclosure__headline--no-padding {
          your code
     }
}
```